corehq/pillows/mappings/app_mapping.py

```diff
@@ -49,6 +49,7 @@
                 'copy_history': {'type': 'string'},
                 'copy_of': {'type': 'string'},
                 'cp_is_active': {'type': 'boolean'},
+                'created_from_template': {'type': 'string'},
```

Comment:
@dannyroberts we could add a step to deploy that at least checks the hash against an MD5 or whatever of the mapping.